.github/ISSUE_TEMPLATE/01-bug.yml

```diff
@@ -0,0 +1,73 @@
+name: Bug report
```

Comment:
The files are always show in an alphanumerical order so we prefix it with 01 to make sure its the first one.